vividus-tests/src/main/resources/story/integration/SetContextStepsTests.story

```diff
@@ -49,23 +49,23 @@ When I switch to frame located `id(exampleCom)`
 When I click on element located by `xpath(//a)`
 
 
-Scenario: Verify steps: "When I reset context" AND "When I change context to element located `$locator`"
-When I change context to element located `By.xpath(//body)`
+Scenario: Verify steps: "When I reset context" AND "When I change context to element located by `$locator`"
```

Comment:
is at least one test for the deprecated syntax kept?

Reply:
story/integration/Accessibility.story:9 - with reset
story/integration/SetContextStepsTests.story:62 - with current context